inference-engine/src/vpu/myriad_plugin/myriad_plugin.cpp

```diff
@@ -111,12 +111,11 @@ Engine::Engine(std::shared_ptr<IMvnc> mvnc) :
     _pluginName = "MYRIAD";
 
     _config = {
-        { KEY_VPU_HW_STAGES_OPTIMIZATION, "ON" },
+        { MYRIAD_ENABLE_HW_ACCELERATION, "ON" },
```

Comment:
Could we use `YES`, `NO` everywhere instead of somewhere `ON`/`OFF` and somewhere `YES`/`NO`?

Reply:
Done